src/fragments/gen2/quickstart/create-nuxtjs-project.mdx

```diff
@@ -0,0 +1,16 @@
+## Create a project
+
+First, you will need to create a new Nuxt app. The following command will create a Nuxt 3 app in a directory called `nuxt-amplify-gen2`.
+
+```bash title="Terminal" showLineNumbers={false}
+npx nuxi@latest init nuxt-amplify-gen2
+
+Need to install the following packages:
+  nuxi@3.10.0
```

Comment:
I wonder if you should include this, because this version will always change, with new updates